starter-code/main.js

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
Your are loading this burger menu file.js, but your dont have any "burger menu" in the html files.
That means that you are trying to get the elements an adding event listeners to elements that doesn't exists.